app/Http/Livewire/LdapSettingsForm.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use App\Models\Ldap;
+use App\Models\Setting;
+use App\Models\Group;
+use Livewire\Component;
+use Illuminate\Support\Facades\Crypt;
+use Illuminate\Support\Facades\Validator;
+
+class LdapSettingsForm extends Component
+{
+      public bool   $ldap_enabled;
+      public        $ldap_server;
+      public bool   $ldap_server_cert_ignore;
+      public        $ldap_uname;
+      public        $ldap_pword;
+      public        $ldap_basedn;
+      public        $ldap_filter;
+      public        $ldap_username_field;
+      public        $ldap_lname_field;
+      public        $ldap_fname_field;
+      public        $ldap_auth_filter_query;
+      public        $ldap_version;
+      public        $ldap_active_flag;
+      public        $ldap_emp_num;
+      public        $ldap_email;
+      public        $ldap_manager;
+      public        $ad_domain;
+      public bool   $is_ad;
+      public bool   $ldap_tls;
+      public bool   $ldap_pw_sync;
+      public        $ldap_jobtitle;
+      public        $ldap_country;
+      public        $ldap_dept;
+      public        $ldap_client_tls_cert;
+      public        $ldap_client_tls_key;
+      public        $ldap_default_group;
+      public        $ldap_phone_field;
+      public        $groups;
+      public        $ldaptest_user;
+      public        $ldaptest_password;
+      public        $ldap_sync_test_users;
+      public        $keys = [];
+      public        $ldap_message;
+      public        $test_login;
+      public        $custom_forgot_pass_url;
+
+
+
+
+    protected $rules = [
+          'ldap_username_field' => 'not_in:sAMAccountName',
+          'ldap_auth_filter_query' => 'not_in:uid=samaccountname|required_if:ldap_enabled,1',
```

Comment:
I'm not sure this is [the right way](https://laravel.com/docs/8.x/validation#rule-not-in) to use `not_in`?

Reply:
they seem to be working  
filter query:
<img width="1080" alt="image" src="https://github.com/snipe/snipe-it/assets/47435081/01149025-73df-4d4b-a635-d44c2e248075">

Reply:
they seem to be working as intended 
<img width="1080" alt="image" src="https://github.com/snipe/snipe-it/assets/47435081/272c4301-12b9-4565-9f91-cf563ad625fc">

Reply:
<img width="1080" alt="image" src="https://github.com/snipe/snipe-it/assets/47435081/7b610df9-7024-4d13-bab4-5597dac0a07c">